docs/_markdown-m1-cocoapods.mdx

```diff
@@ -0,0 +1,10 @@
+<details>
+  <summary>Note for Mac M1 users</summary>
+Mac M1 architecture is not directly compatible with Cocoapods. If you encounter issues when installing pods, you can solve it by running:
+
+- `sudo arch -x86_64 gem install ffi`
+- `arch -x86_64 pod install`
+
+These commands install the `ffi` package, to load dynamically-linked libraries and let you run the `pod install` properly, and runs `pod install` with the proper architecture.
+
+</details>
```

Comment:
I would use an info admonitions here:
https://docusaurus.io/docs/next/markdown-features/admonitions instead of the `<details>` block

Reply:
It was my first, preferred approach to, but it looks like summary+details looks quite nice in there, plus I like the collapsible part, this information affects only low percentage of the RN users.

Ohh, I want to provide the screenshot from the preview, but it has already been updated. 😅

Reply:
Awesome let's stick to this then 👍

Reply:
@cipolleschi could you revert to use <details>/<summary> (sorry for the long rtt)?

Reply:
@cortinico: done! @Simek: reverted. If it's ok, we can merge it! :D 
Thank you a lot for the review.